### PR TITLE
Added AVAILABLE_LOCALES env var in ofn-install

### PR DESCRIPTION
**Pull request by mllocs**
Added the AVAILABLE_LOCALES env var in the application.yml template so it can be defined in every instance. I also set the available_locales to "es,ca,pt" in Katuma (es.yml).

 - [x] Test without `available_locales` to see what happens (OR should work)

### CODE REVIEW

File: roles/app/templates/application.yml.j2 [hunk 1]
```diff
@@ -9,6 +9,7 @@ TIMEZONE: {{ timezone }}
 DEFAULT_COUNTRY_CODE: {{ country_code }}
 # Locale for translation.
 LOCALE: {{ locale }}
+AVAILABLE_LOCALES: {{ available_locales }}
```
Comment (enricostano) [inline roles/app/templates/application.yml.j2]: What about group vars that don't specify `available_locales`?

**Comment by mllocs** [comment]
so, if we want english in Katuma we need to add it here https://github.com/openfoodfoundation/ofn-install/pull/123/files#diff-cf20333b737bfac9bf3fece3370e7a3eR7

**Comment by sauloperez** [comment]
I believe so

**Comment by enricostano** [comment]
I tested this in our staging, it works. Many :heart: to @mllocs

**Comment by mllocs** [comment]
boaaaaaaaaaaaa

**Comment by sauloperez** [comment]
Is that portuguese :trollface:?

**Comment by mllocs** [comment]
screen shot 2018-03-14 at 15 59 42

**Comment by sauloperez** [comment]
Lol

**Pull request merged**